lib/deploy.go

```diff
@@ -15,17 +16,49 @@ import (
 // and then deploys the package to salesforce
 func PushByPaths(fpaths []string, byName bool, namePaths map[string]string, opts *ForceDeployOptions) {
 	pb := NewPushBuilder()
-
+	fmt.Println("directory")
```

Comment:
Is this just for debugging?

Reply:
Yea, will be removed.

Reply:
@dcarroll  these debugging stuff have been removed by next commit.